docs/beginner/tutorial2-surface/README.md

```diff
@@ -223,13 +223,35 @@ Now that we've configured our surface properly we can add these new fields at th
 
 Since our `State::new()` method is async we need to change `run()` to be async as well so that we can await it.
 
+Our `window` has beened moved to the State instance, we will need to update our `event_loop` to reflect this.
+
 ```rust
 pub async fn run() {
```

Comment:
Something I missed was that this line needs `#[cfg_attr(target_arch = "wasm32", wasm_bindgen(start))]` before it.

Reply:
Added in 80656c7